app/views/take_whole_pot_calculator/show.html.erb

```diff
@@ -0,0 +1,83 @@
+<% content_for(:data_layer) do %>
```

Comment:
Moved in eac16ac4340051a3a1678cd9b8894356738d341d. Tried rewriting the commit history but it got messy.